htdocs/web_portal/index.php

```diff
@@ -686,6 +712,25 @@ function Get_Static_Page_Contents($Page_Name) {
 }
 
 
+/* Get the contents of the static PHP page specified in $PHP_Page_Name
+ * if the page name isn't specified then return a blank string */
+function Get_Static_PHP_Page_Contents($PHP_Page_Name) {
+    require_once __DIR__.'/components/Draw_Components/draw_page_components.php';
+    $phpDir = __DIR__."/static_php";
+    $Available_Static_Pages = Get_Directory_Contents($phpDir);
+    if(!isset($Available_Static_Pages[$PHP_Page_Name])) {
+        return "";
+    }
+    $PHP = Get_File_Contents($phpDir."/".$PHP_Page_Name);
```

Comment:
Here's the problem (line 724) - Get_File_Contents just gets the contents of the file, it doesn't execute it. Therefore the php parts aren't being executed, just written out. idk what to replace it with though

Reply:
Fixed this - I just used the commented out statements below I'd tried before! I had to use the `systemctl reload httpd` to update it, which is maybe why it hadn't worked before.